src/parser.c

```diff
@@ -301,11 +301,11 @@ static void parseHeaderLine(flightLog_t *log, mmapStream_t *stream)
         uint32_t u;
     } floatConvert;
 
-    if (streamPeekChar(stream) != ' ')
+    if (streamReadByte(stream) != 'H')
```

Comment:
Please always use blocks after `if`	/ `for`. This will lead to much better maintainable code.